dev/breeze/pyproject.toml

```diff
@@ -69,7 +69,7 @@ dependencies = [
     "pyyaml>=6.0.2",
     "requests>=2.31.0",
     "restructuredtext-lint>=1.4.0",
-    "rich-click>=1.7.1",
+    "rich-click>=1.8.9",
```

Comment:
Should we try something like 

```suggestion
    "rich-click>=1.7.1, !=1.8.9",
```

Reply:
right, but I'm still a bit confused what's happening 🤔

Reply:
I am also running `uv tool install -e ./dev/breeze --force` on main once.

Reply:
It was failing for 1.7.1 for me